sanic/app.py

```diff
@@ -1146,7 +1146,7 @@ async def create_server(
             ssl=ssl,
             sock=sock,
             unix=unix,
-            loop=get_event_loop(),
+            use_existing_loop=True,
```

Comment:
This is a breaking change on a public API. We need to keep the same outward facing API on this.

Reply:
We can think about changing it to something like this. But ultimately, if someone does:

```
create_server(..., loop=loop)
```

We still need to support that. Meaning it would have to override `use_existing_loop`.

Reply:
Unless there is a compelling reason, I would lean towards tabelling this change until v22.

Reply:
The `create_server` function didn't/doesn't have a `loop` parameter, and the `_helper` function isn't a public API. Am I missing something here?

Reply:
![image](https://user-images.githubusercontent.com/166269/140702051-d20c7536-6221-4d2a-a97f-f3484182a5a2.png)

Sorry, I did not expand the view to see that this was not in the signature